tests/expectedoutputs/TestWCS/test_WCSGetCapabilities_testdatanc

```diff
@@ -22,36 +22,36 @@
     <GetCapabilities>
       <DCPType>
         <HTTP>
-          <Get><OnlineResource xlink:type="simple" xlink:href="%26source%3Dtestdata%2Enc&amp;SERVICE=WCS&amp;" /></Get>
+          <Get><OnlineResource xlink:type="simple" xlink:href="source=testdata%2Enc&amp;SERVICE=WCS&amp;" /></Get>
```

Comment:
All the test result changes seem to be dropping the leading `&`. Is that intended?

Reply:
Yes. Has to do with the fix for over-eagerly urlencoding.